ewcmd.py

```diff
@@ -224,7 +224,8 @@ def gen_data_text(
 		for cosmetic in cosmetics:
 			cos = EwItem(id_item = cosmetic.get('id_item'))
 			if cos.item_props['adorned'] == 'true':
-				adorned_cosmetics.append(cosmetic.get('name'))
+				hue = ewcfg.hue_map.get(cos.item_props.get('hue'))
+				adorned_cosmetics.append((hue.str_name + " colored " if hue != None else "") + cosmetic.get('name'))
```

Comment:
we should make this change for when you !data other people, too. see gen_data_text()

Reply:
done